Formula/gh.rb

```diff
@@ -14,13 +15,9 @@ class Gh < Formula
   depends_on "go" => :build
 
   def install
-    ldflags = %W[
-      -X github.com/cli/cli/command.Version=#{version}
-      -X github.com/cli/cli/command.BuildDate=#{Date.today}
-      -s -w
```

Comment:
Probably we want to keep `-s -w` LDFLAGS

Reply:
Well, the official formula doesn't have them https://github.com/github/homebrew-gh/blob/ad8497a/Formula/gh.rb but I can pass them through `LDFLAGS` make variable? 🤔 
Btw non-brew builds seem to be built with `-s -w` https://github.com/cli/cli/blob/v0.9.0/.goreleaser.yml#L11-L20 🙃

Reply:
> I can pass them through `LDFLAGS` make variable? 🤔

Yep, let's do it

Reply:
Done